### ISSUE TITLE
Building from source fails because src folder doesn't exist

### ISSUE THREAD

**Issue by jariz** [mapbox/node-or-tools#34]
The src folder exists in the repo but not in the npm package (it seems).  
It's explicitly ignored in `.npmignore`, why?

**Environment**  
OSX/Linux (tried both)

**Reproducible by**
run `npm install node_or_tools --build-from-source`

**Expectation**
Building from source succeeds.

**Logs**  
OSX: ([full](https://gist.github.com/jariz/15378d4e91d4fd00009da34baeeaae43))
```
  TOUCH Release/obj.target/action_before_build.stamp
  SOLINK_MODULE(target) node_modules/node_or_tools/lib/binding/node_or_tools.node
clang: error: no such file or directory: 'Release/obj.target/node_or_tools/src/main.o'
clang: error: no such file or directory: 'Release/obj.target/node_or_tools/src/tsp.o'
clang: error: no such file or directory: 'Release/obj.target/node_or_tools/src/vrp.o'
make: *** [node_modules/node_or_tools/lib/binding/node_or_tools.node] Error 1
```

Linux: ([full](https://gist.github.com/jariz/449ceecf883a434ef7bd8a94644769db))
```
  TOUCH Release/obj.target/action_before_build.stamp
  SOLINK_MODULE(target) /usr/src/app/node_modules/node_or_tools/lib/binding/node_or_tools.node
g++: error: Release/obj.target/node_or_tools/src/main.o: No such file or directory
g++: error: Release/obj.target/node_or_tools/src/tsp.o: No such file or directory
g++: error: Release/obj.target/node_or_tools/src/vrp.o: No such file or directory
make: *** [/usr/src/app/node_modules/node_or_tools/lib/binding/node_or_tools.node] Error 1
node_or_tools.target.mk:156: recipe for target '/usr/src/app/node_modules/node_or_tools/lib/binding/node_or_tools.node' failed
```

**Comment by jariz** [comment]
Cloning this repo and then running `npm i` does, in fact work.  
But when using node_or_tools as a dependency, compilation will fail.

**Comment by daniel-j-h** [comment]
Probably an oversight because we use the pre-built binaries for node 4 and node 6 (lts versions). Feel free to make a pr adding the source directory to the npmignore whitelist (and the deps file in the scripts directory, too).

**Comment by jariz** [comment]
Fixed by #36

**Comment by daniel-j-h** [comment]
Just published v1.0.5 including the changes.

Sorry for the regression based on https://github.com/mapbox/node-or-tools/issues/32#issuecomment-320662762.

**Comment by jariz** [comment]
Thanks dude!
